tests/yam/agama/boot_agama.pm

```diff
@@ -32,6 +32,10 @@ sub post_fail_hook {
 sub run {
     my $self = shift;
 
+    if (my $agama_profile = get_var('AGAMA_PROFILE')) {
+        my $path = expand_agama_profile($agama_profile);
+        set_var('AGAMA_PROFILE', $path);
+    }
```

Comment:
please move this section after line 65 and add a comment here:
`Please, avoid adding code here that would be a dependency for specific booting implementations`

Reply:
updated, then no remote worker supported.